internal/services/automation/automation_software_update_configuration.go

```diff
@@ -512,12 +513,20 @@ func (m SoftwareUpdateConfigurationResource) Arguments() map[string]*pluginsdk.S
 					"classification_included": {
```

Comment:
As this is supposed to be a list, and this property is misnamed can we deprecate this and create a new property `classifications_included` that takes a list and translates it to a CSV list for the api?

Reply:
I have tried with a new property but we have to make it compatible with the deprecated one or there will be diff in the `terraform plan`. So I have to set both properties as `Computed: true` to ignore the diff. I don't think it's a so good solution. Is there better practice for this scenario?

Reply:
yes you will need to set both to computed, always set both, make them conflict, and the use the one that is set. that is the right way to deprecate.